### ISSUE TITLE
Kolibri-gnome fails at Soup.Message does not have attribute 'status_code'

### ISSUE THREAD

**Issue by starnight** [learningequality/kolibri-installer-gnome#94]
I run org.learningequality.Kolibri flatpak for the first time on a clean & fresh Endless OS.  But, it crashes and shows error:
```
$ flatpak run org.learningequality.Kolibri 
INFO: No C extensions are available for this platform
INFO: 
INFO: ************************************
INFO: *  Kolibri GNOME App Initializing  *
INFO: ************************************
INFO: 
INFO: Started at: 2023-08-31 10:57:37.578765
Failed to create /sysroot/home/dev/.var/app/org.learningequality.Kolibri/cache for shader cache (No such file or directory)---disabling.
Failed to create /sysroot/home/dev/.var/app/org.learningequality.Kolibri/cache for shader cache (No such file or directory)---disabling.
INFO: Provisioning device…
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/kolibri_gnome/kolibri_daemon_manager.py", line 185, in __kolibri_api_get_async_on_soup_send
    f"Error calling Kolibri API, code: {soup_message.status_code}"
AttributeError: 'Message' object has no attribute 'status_code'
```

**Comment by starnight** [comment]
Here is the flatpak informaiton:
```
$ flatpak info org.learningequality.Kolibri 

Kolibri - Offline education technology platform

          ID: org.learningequality.Kolibri
         Ref: app/org.learningequality.Kolibri/x86_64/beta
        Arch: x86_64
      Branch: beta
     Version: 2.3.3
     License: MIT
      Origin: flathub-beta
  Collection: org.flathub.Beta
Installation: system
   Installed: 173.1 MB
     Runtime: org.gnome.Platform/x86_64/44
         Sdk: org.gnome.Sdk/x86_64/44

      Commit: f7526342b610bfd653ca2497183d643eac10e11d3ba8f363015be0d06fa29c0d
      Parent: 6039fa0d5646ba8d22cc801c711929bc7669ac587c138fe6e05b66032172d024
     Subject: Update kolibri-gnome to v2.3.3 (018e780f)
        Date: 2023-08-30 21:20:21 +0000
```

**Comment by starnight** [comment]
Hmm ...  It is beta version.

After uninstall the beta version and reinstall the stable version from Flathub.  It works!
```
$ flatpak info org.learningequality.Kolibri 

Kolibri - Offline education technology platform

          ID: org.learningequality.Kolibri
         Ref: app/org.learningequality.Kolibri/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 2.3.3
     License: MIT
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 173.1 MB
     Runtime: org.gnome.Platform/x86_64/44
         Sdk: org.gnome.Sdk/x86_64/44

      Commit: 73f997e7781c1ff2dbe75df9c941e851fca4d432e463bb2ad8b1ac5b895c7fe4
      Parent: 1a61f22d76c1f569dd670fadfedcac9ea5525a6f78fc4cd9c657ef68fe480bd2
     Subject: Update kolibri-gnome to v2.3.3 (70d07c90)
        Date: 2023-08-30 21:47:27 +0000
```

**Comment by starnight** [comment]
$ flatpak info org.learningequality.Kolibri 

Kolibri - Offline education technology platform

          ID: org.learningequality.Kolibri
         Ref: app/org.learningequality.Kolibri/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 2.3.3
     License: MIT
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 173.1 MB
     Runtime: org.gnome.Platform/x86_64/44
         Sdk: org.gnome.Sdk/x86_64/44

      Commit: 73f997e7781c1ff2dbe75df9c941e851fca4d432e463bb2ad8b1ac5b895c7fe4
      Parent: 1a61f22d76c1f569dd670fadfedcac9ea5525a6f78fc4cd9c657ef68fe480bd2
     Subject: Update kolibri-gnome to v2.3.3 (70d07c90)
        Date: 2023-08-30 21:47:27 +0000

**Comment by dylanmccall** [comment]
> Hmm ... It is beta version. After I uninstall the beta version and reinstall the stable version, it works

Yes, that beta branch was a mess for a while. I'm surprised that particular update didn't reach you from the beta branch, but maybe just a funny caching / timing issue :)

I reorganized it yesterday, and I made a ticket to formalize how releases happen for that flatpak over here: flathub/org.learningequality.Kolibri#69. Especially now that we're looking at this thing more closely, it'll be important that we can rely on our test builds!
